packages/common/src/components/FormGroupWithHelpText/FormGroupWithHelpText.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import {
+  FormGroup,
+  FormGroupProps,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+} from '@patternfly/react-core';
+
+export interface FormGroupWithHelpTextProps extends FormGroupProps {
+  /**
+   * Sets the FormGroup validated. If you set to success, text color of helper text will be modified to indicate valid state.
+   * If set to error, text color of helper text will be modified to indicate error state.
+   * If set to warning, text color of helper text will be modified to indicate warning state.
+   */
+  validated?: 'success' | 'warning' | 'error' | 'default';
+  /**
+   * Helper text regarding the field. It can be a simple text or an object.
+   */
+  helperText?: React.ReactNode;
+  /**
+   * Helper text after the field when the field is invalid. It can be a simple text or an object.
+   */
+  helperTextInvalid?: React.ReactNode;
+}
+
+/**
+ *  A FormGroup component that supports helperTexts
+ *
+ *  This component wraps the FormGroup with an option to use the following helper text related properties
+ *  (since not supported anymore as part of the FormGroup component in PatternFly 5):
+ *  helperText, helperTextInvalid, validated
+ *
+ * `See` https://www.patternfly.org/get-started/release-highlights/#helper-text
+ *
+ * [<img src="static/media/src/components-stories/assets/github-logo.svg"><i class="fi fi-brands-github">
+ * <font color="green">View component source on GitHub</font>](https://github.com/kubev2v/forklift-console-plugin/blob/main/packages/common/src/components/FormGroupWithHelpText/FormGroupWithHelpText.tsx)
+ */
+export const FormGroupWithHelpText: React.FC<FormGroupWithHelpTextProps> = ({
+  label,
+  isRequired,
+  fieldId,
+  labelIcon,
+  role,
+  children,
+  validated,
+  helperText,
+  helperTextInvalid,
+}) => {
+  const validatedToVariant = (validated) =>
+    !validated || validated === 'default' ? 'indeterminate' : validated;
+  const helperTextMsg = validated === 'error' ? helperTextInvalid : helperText;
```

Comment:
question, what is the behavior when `validated === 'error'` and  `helperTextInvalid` is undefined ?

Reply:
In such a case, no help text message will be displayed. The same behavior as if using the PF4 `FormGroup` in that same case (`validated`='error', `helperTextInvalid`=undefined)

Reply:
hmm, lets make our version show the regular `helperText`, instead, it makes more sense to me to fall back into the regular helper text than to show nothing ...